cms/envs/aws.py

```diff
@@ -143,6 +143,7 @@
 
 SESSION_COOKIE_DOMAIN = ENV_TOKENS.get('SESSION_COOKIE_DOMAIN')
 SESSION_COOKIE_HTTPONLY = ENV_TOKENS.get('SESSION_COOKIE_HTTPONLY', True)
+SESSION_SAVE_EVERY_REQUEST = ENV_TOKENS.get('SESSION_SAVE_EVERY_REQUEST', False)
```

Comment:
Reviewer's note: this will effectively configure a new django setting that we had not used yet, and set it to its django default. I did check that the default remains unchanged in Django 1.8.